src/components/ScholarshipDetailCardCell.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles, Grid, Typography } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    color: 'rgba(0, 0, 0, 0.6)',
+    padding: `${theme.spacing(1.5)}px 0`,
+    borderBottom: '1px solid #EEEEEE',
+  },
+  labelWrapper: {
+    '& p': {
+      fontWeight: 'normal',
+    },
+  },
+  textWrapper: {
+    '& p': {
+      textAlign: 'right',
+      fontWeight: '500',
+    },
+  },
+}));
+
+const ScholarshiDetailCardCell = ({ label, text }) => {
```

Comment:
*ScholarshipDetailCardCell

Reply:
Might be better to keep this component inside of ScholarshipDetailCard since it's only used there. If this component is useful in another component later we can refactor it and give it a more generic name.